HIP/hip-0000-virtual-merkle.md

```diff
@@ -0,0 +1,150 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Virtual Merkle
+- author: Richard Bair (@rbair23), Jasper Potts (@jasperpotts)
+- type: Standards Track
+- category: Core
+- status: Draft
+- created: 2021-09-07
+- discussions-to: <a URL pointing to the official discussion thread>
```

Comment:
Please link to this Discussion thread: https://github.com/hashgraph/hedera-improvement-proposal/discussions/139